spec/new-productions.htm

```diff
@@ -9,6 +9,18 @@
   </emu-grammar>
 </emu-clause>
 
+<emu-clause id="public-field-definition-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+    PublicFieldDefinition :
+      PropertyName[?Yield] Initializer?
+  <emu-grammar>
+  </emu-grammar>
+  <emu-alg>
+    1. It is a SyntaxError if |Initializer| contains |NewTarget|.
```

Comment:
In the current state, this and the below line are poorly enough specified that I would object to the proposal's advancement. I legitimately do not know what is meant by each of them.